<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>419</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>452</width>
    <height>419</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>452</width>
    <height>419</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Reverse</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>170</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(74, 74, 74);
border-color: rgb(225, 225, 225);


</string>
    </property>
    <widget class="QRadioButton" name="encryption_radioButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>60</y>
       <width>130</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(247, 247, 247)</string>
     </property>
     <property name="text">
      <string>Зашифровать</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="decryption_radioButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>100</y>
       <width>130</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Расшифровать</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>170</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(235, 235, 235);
border-width: 2px;
border-style: solid;</string>
     </property>
     <property name="text">
      <string>Действие</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>170</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(74, 74, 74);
border-color: rgb(225, 225, 225);</string>
    </property>
    <widget class="QRadioButton" name="RU_radioButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(247, 247, 247)</string>
     </property>
     <property name="text">
      <string>RU</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="EU_radioButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(247, 247, 247)</string>
     </property>
     <property name="text">
      <string>EU</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>170</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(235, 235, 235);
border-width: 2px;
border-style: solid;</string>
     </property>
     <property name="text">
      <string>Язык текста</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>170</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(247, 247, 247);
border-width: 2px;
border-style: solid;
border-color: rgb(243, 243, 243);
background-color: rgb(74, 74, 74);</string>
    </property>
    <property name="text">
     <string>Итог</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(247, 247, 247);
border-width: 2px;
border-style: solid;
border-color: rgb(243, 243, 243);
background-color: rgb(74, 74, 74);</string>
    </property>
    <property name="text">
     <string>Введите текст</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>365</y>
      <width>250</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(74, 74, 74);
border-color: rgb(225, 225, 225);
border-width: 2px;
border-style: solid;
border-radius: 10px;
color: rgb(235, 235, 235);
</string>
    </property>
    <property name="text">
     <string>Получить результат</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="input_textEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>170</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>460</width>
      <height>420</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>460</width>
      <height>420</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>460</width>
      <height>420</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/123.png</pixmap>
    </property>
   </widget>
   <widget class="QTextEdit" name="output_textEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>170</width>
      <height>71</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>170</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(247, 247, 247);
border-width: 1px;
border-style: solid;
border-color: rgb(243, 243, 243);
background-color: rgb(74, 74, 74);</string>
     </property>
     <property name="text">
      <string>Длинна группы</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="kayBox">
     <property name="geometry">
      <rect>
       <x>118</x>
       <y>10</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(247, 247, 247);
border-width: 1px;
border-style: solid;
border-color: rgb(243, 243, 243);
background-color: rgb(74, 74, 74);</string>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(247, 247, 247);
border-width: 1px;
border-style: solid;
border-color: rgb(243, 243, 243);
background-color: rgb(74, 74, 74);</string>
     </property>
     <property name="text">
      <string>Символов</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="kayBox_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(247, 247, 247);
border-width: 1px;
border-style: solid;
border-color: rgb(243, 243, 243);
background-color: rgb(74, 74, 74);</string>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </widget>
   <zorder>label_6</zorder>
   <zorder>widget</zorder>
   <zorder>widget_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>pushButton</zorder>
   <zorder>input_textEdit</zorder>
   <zorder>output_textEdit</zorder>
   <zorder>widget_3</zorder>
   <zorder>widget_4</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
